src/main/java/com/crowdin/client/Sandbox.java

```diff
@@ -12,17 +13,33 @@
 public class Sandbox {
 
     public static void main(String[] args) throws FileNotFoundException, InterruptedException {
-        if (args.length > 0) {
-            try {
-                Credentials credentials = new Credentials(args[0], "oliynyk");
-                var client = new Client(credentials);
-                ResponseObject<User> user = client.getUsersApi().getAuthenticatedUser();
-                System.out.println(user.getData());
-            } catch (HttpException e) {
-                System.out.println(e.getError());
-            } catch (HttpBadRequestException e) {
-                System.out.println(e.getErrors());
-            }
-        }
+
+        new Integer(null);
+
+        if (true) return;
+
+        Credentials credentials = new Credentials("1af38c02dd24a61fa721cc77e099ff9508f8e58b8417ba040b18507e5f5cbffe4ddcdcb84e5610c7", null);
+        Client client = new Client(credentials);
+
+        ProjectSettings project = (ProjectSettings) client.getProjectsGroupsApi()
+            .getProject(383090L)
+            .getData();
+
+        System.out.println("project = " + project);
+
+        if (true) return;
+
+//        if (args.length > 0) {
```

Comment:
???

Reply:
oh, it's a sandbox. Maybe it will be better to add this sandbox to the .gitignore?

Reply:
Oh, didn't notice how I committed it.

Reply:
It's even worse. I committed the wrong file.